cloud/scope/vnic_reconciler.go

```diff
@@ -65,7 +65,7 @@ func (m *MachineScope) createVnicAttachment(ctx context.Context, spec infrastruc
 		}
 	}
 
-	tags := m.getFreeFormTags(*m.OCICluster)
+	tags := m.getFreeFormTags()
```

Comment:
I may have missed this why isn't the cluster being passed?

Reply:
we changed impl in machine.go getFreeFormTags method to use cluster accessor